RELEASES.md

```diff
@@ -0,0 +1,36 @@
+# Fedora CoreOS Releases
+
+FCOS releases normally happen every 2 weeks. In addition,
+there may be asynchronous releases for e.g. CVEs or major
+regressions.
```

Comment:
```suggestion
there may be asynchronous releases for e.g. CVEs or bug fixes.
```

Since we may spin `testing` for non-major regressions let's be a little more ambiguous here.